### PR TITLE
SWF-3534: Update Release Notes and Changelog for 4.3.0

### PR DESCRIPTION
**Pull request by mgreau**
I start this PR to discuss about all informations to update for PLF 4.3.0.
For now I have just started by updating the 4.3.0-RC1 version to 4.3.0.

@minhdv Please let me know how and where to find all the required informations. Thanks.

### CODE REVIEW

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt [hunk 1]
```diff
@@ -25,23 +26,29 @@ eXo Platform 4.3.0-RC1
     * [PLF-6459] - [Registration] [Jboss] Register software feature can not run on java8
     * [PLF-6460] - [Registration] No error message displays when error happens during this last register call,
     * [PLF-6478] - Preview image on activity stream not fully responsive
+    * [PLF-6484] - Cannot create a user an error is occured
```
Comment (trangvh) [inline plf-packaging-resources/src/main/resources/CHANGE_LOG.txt]: should remove, it's resolved as Duplicate

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt [hunk 2]
```diff
@@ -25,23 +26,29 @@ eXo Platform 4.3.0-RC1
...
+    * [PLF-6510] - [Java 8] No chart of Non-Heap Memory
+    * [PLF-6512] - [acme] Unfriendly categories on Products page
+    * [PLF-6514] - Failed to instanciate plugin UpgradeUserNotificationSettingPlugin with profile minimal
+    * [PLF-6534] - [Java 8][Cluster] Cannot start tomcat package with cluster mode on java 8
 
 ** Feedback
     * [PLF-5281] - Simpler article content type
 
 ** Improvement
     * [PLF-6422] - Upgrade to JCR 1.16.2-GA
```
Comment (trangvh) [inline plf-packaging-resources/src/main/resources/CHANGE_LOG.txt:38]: should remove

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt [hunk 3]
```diff
@@ -97,6 +104,42 @@ eXo ECMS 4.3.0-RC1
...
+    * [ECMS-7320] - [IE 11] Problem when mouse over on CLV portlet
+    * [ECMS-7321] - Content is not displayed when input java script
+    * [ECMS-7322] - Publication statuses [Pending/Approved/Staged] do not display in edit mode
+    * [ECMS-7327] - HTML tag is added when edit the Illustrated web content
+    * [ECMS-7329] - [Site Explorer] Cannot upload file which has the same name with folder
+    * [ECMS-7333] - [CLV] Cannot [edit in content] when the content is in draft
+    * [ECMS-7334] - [WCM] Cannot edit content of some content types directly in CLV/SCV portlet
+    * [ECMS-7336] - [Open in Office] Missing icon in action bar when tick on document
+    * [ECMS-7338] - [Open in Office] NOT display icon "Open in" to open file with read permission user
+    * [ECMS-7339] - [Open in Office] File is always locked by john
```
Comment (trangvh) [inline plf-packaging-resources/src/main/resources/CHANGE_LOG.txt]: to remove

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt [hunk 4]
```diff
@@ -97,6 +104,42 @@ eXo ECMS 4.3.0-RC1
...
+    * [ECMS-7295] - [Document Versioning] The checkbox "Remember my choice" is not considered for Cancel action
+    * [ECMS-7296] - [Document Versioning] Alert message is not displayed when upload the same document from the Activity Selector
+    * [ECMS-7298] - [WCM Template pack] Cannot deploy some links
+    * [ECMS-7300] - [Document Preview] Error happens when like document in Preview mode
+    * [ECMS-7303] - [ECMS]Not save the content of a webcontent after created
+    * [ECMS-7304] - Wrong name of File content when creating at the root of a drive
+    * [ECMS-7305] - [ECMS] Not show new template after created
+    * [ECMS-7309] - [Remote edit]Doesn't work in personal drive
+    * [ECMS-7310] - [ECMS] Cannot edit data of upload file
+    * [ECMS-7312] - [IE11] Background of ACME Signin portlet is not changed after creating a css file in Share folder
```
Comment (trangvh) [inline plf-packaging-resources/src/main/resources/CHANGE_LOG.txt]: to remove

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt [hunk 5]
```diff
@@ -269,16 +355,25 @@ eXo Calendar 4.3.0-RC1
     * [CAL-1147] - No reminder pop-up when Platform uses Apache web server
```
Comment (trangvh) [inline plf-packaging-resources/src/main/resources/CHANGE_LOG.txt]: to remove (no fix)

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt [hunk 6]
```diff
@@ -287,19 +382,25 @@ eXo Integration 4.3.0-RC1
...
+    * [INTEG-448] - [integ-ecms] Failed to load resource bundles for File/Content Activities
 
 
-eXo GateIn 4.3.0-RC1-PLF  
+eXo GateIn 4.3.0-PLF
 *************************
 
 ** Bug
     * [EXOGTN-1416] - Composite CSS file of portlet in portals is not build properly
+    * [EXOGTN-1750] - [Safari-ipad] Impossible to Drag-Drop portlet when creating/editing a page
```
Comment (trangvh) [inline plf-packaging-resources/src/main/resources/CHANGE_LOG.txt]: to remove (Cannot Reproduce)

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt [hunk 7]
```diff
@@ -310,6 +411,16 @@ eXo GateIn 4.3.0-RC1-PLF
     * [EXOGTN-2107] - Send button is not disabled when the field Username or Email address is not input
     * [EXOGTN-2110] - Sender info is not matched with Notification definition
     * [EXOGTN-2114] - [Forgot Password] Cannot start Jboss package with Java 8
+    * [EXOGTN-2124] - The text is wrong of success noticication in User settings
+    * [EXOGTN-2127] - [Java8][OpenAM] Cannot install OpenAM configuration with java8
```
Comment (trangvh) [inline plf-packaging-resources/src/main/resources/CHANGE_LOG.txt]: to remove, it's in Known issues

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt [hunk 8]
```diff
@@ -310,6 +411,16 @@ eXo GateIn 4.3.0-RC1-PLF
...
+    * [EXOGTN-2129] - [Oauth - Facebook Authentication - API v2.5] Information of the Facebook user is not extracted
+    * [EXOGTN-2136] - [Oauth Authentication] The avatar is not extracted
+    * [EXOGTN-2153] - [Oauth] Username extracted from the social network matches an existing eXo user name causes exception
+    * [EXOGTN-2154] - [Oauth] Must type two times to login via linkedin
```
Comment (trangvh) [inline plf-packaging-resources/src/main/resources/CHANGE_LOG.txt]: to remove

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt [hunk 9]
```diff
@@ -310,6 +411,16 @@ eXo GateIn 4.3.0-RC1-PLF
...
+    * [EXOGTN-2155] - [Restricted Page Editor] User is not granted can move apps and move containers
```
Comment (trangvh) [inline plf-packaging-resources/src/main/resources/CHANGE_LOG.txt]: to remove

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt [hunk 10]
```diff
@@ -287,19 +382,25 @@ eXo Integration 4.3.0-RC1
     * [INTEG-426] - wrong position of x icon
     * [INTEG-432] - [Unified Search] Missing content type "Task" in the Filter
     * [INTEG-435] - Bad display of Recent Activities in my profil
+    * [INTEG-436] - [integ-wiki] No activity when creating space wiki page
+    * [INTEG-438] - Failed to instanciate plugin SOC_ACTIVITY_MIGRATION with profile "minimal"
+    * [INTEG-443] - [Responsive Layout] An error is occured after selecting + menu from NavigBar
```
Comment (trangvh) [inline plf-packaging-resources/src/main/resources/CHANGE_LOG.txt]: to remove

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt [hunk 11]
```diff
@@ -287,19 +382,25 @@ eXo Integration 4.3.0-RC1
...
+    * [INTEG-448] - [integ-ecms] Failed to load resource bundles for File/Content Activities
```
Comment (trangvh) [inline plf-packaging-resources/src/main/resources/CHANGE_LOG.txt]: to remove

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt [hunk 12]
```diff
@@ -25,23 +26,29 @@ eXo Platform 4.3.0-RC1
...
     * [PLF-6489] - [Tomcat - PLF4.3] Cannot create a user
     * [PLF-6490] - [Registration][Jboss] Cannot access to login screen
+    * [PLF-6506] - [Space Navigation] Wrong label of "No space found" in MY SPACES
+    * [PLF-6510] - [Java 8] No chart of Non-Heap Memory
+    * [PLF-6512] - [acme] Unfriendly categories on Products page
+    * [PLF-6514] - Failed to instanciate plugin UpgradeUserNotificationSettingPlugin with profile minimal
+    * [PLF-6534] - [Java 8][Cluster] Cannot start tomcat package with cluster mode on java 8
 
 ** Feedback
     * [PLF-5281] - Simpler article content type
```
Comment (trangvh) [inline plf-packaging-resources/src/main/resources/CHANGE_LOG.txt:35]: should remove

**Comment by mgreau** [comment]
PR merged

**Pull request closed without merging**